pkgs/development/python-modules/flask-autoindex/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, flask
+, flask-silk
+, future
+}:
+
+buildPythonPackage rec {
+  pname = "Flask-AutoIndex";
+  version = "0.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "19b10mb1nrqfjyafki6wnrbn8mqi30bbyyiyvp5xssc74pciyfqs";
+  };
+
+  propagatedBuildInputs = [
+    flask
+    flask-silk
+    future
+  ];
+
+  postInstall = ''
+    # binary does not run, known issue and fixed in next release (>0.6)
+    # see https://github.com/sublee/flask-autoindex/pull/32
+    rm $out/bin/fai
```

Comment:
I don't think there will be a next release. Maybe use https://github.com/sublee/flask-autoindex/pull/32.patch as a patch?

Reply:
You might be right. Done.